_data/examples.yml

```diff
@@ -102,3 +102,10 @@
   author: Clara Petroll
   thumbnail: oscar.png
   language: julia
+
+- title: "Tropicalization of a p-adic surface"
+  repository: micjoswig/oscar-notebooks
+  filename: patchworking/Tropicalization
+  author: Michael Joswig
+  thumbnail: oscar.png
```

Comment:
Should this use instead the `polymake.png` logo?

Reply:
This does `using Oscar` and does not just use polymake, so I think it'd be fine to have it use the oscar logo. If @micjoswig changes his mind, it'll be easy enough to change it later. So just merge it as-is for now?

Reply:
I was thinking as much, I will merge later today.